app/shared/modals/reservation-info/ReservationEditForm.js

```diff
@@ -97,9 +100,36 @@ class UnconnectedReservationEditForm extends Component {
 
   renderReservationTime() {
     const {
-      isEditing, reservation, resource, t,
+      isEditing, reservation, resource, t, userUnitRole,
     } = this.props;
+
     if (isEditing) {
+      const canIgnoreOpeningHours = hasPermissionForResource(
+        userUnitRole,
+        resourcePermissionTypes.CAN_IGNORE_OPENING_HOURS,
+      );
+      const reservationDate = moment(reservation.begin).format('YYYY-MM-DD');
+      // We have a generic utility function with the same purpose, but
+      // for some reason the fields on the resource object are in
+      // camelCase when we access them here. The generic utility
+      // function assumes snake_case.
+      const getOpenHours = (resource0, reservationDate0) => {
```

Comment:
the argument with number is kinda bad, how about `getReservationOpeningHoursByDate` ? 😄 
Bring that to method level (same level with `renderReservationTime` or make it generic and bring it to `https://github.com/City-of-Helsinki/varaamo/blob/develop/src/domain/reservation/utils.js` instead of nesting it here